src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -430,20 +434,22 @@ async def set_tpz_io_settings(self, voltage_limit, hub_analog_input):
         else:
             raise ValueError("voltage_limit must be 75 V, 100 V or 150 V")
 
-        payload = st.pack("<HHHHH", 1, voltage_limit, hub_analog_input, 0, 0)
+        payload = st.pack(
+            "<HHHHH", _CHANNEL, voltage_limit, hub_analog_input, _RESERVED, _RESERVED
```

Comment:
```suggestion
            "<HHHHH", Tpz._CHANNEL, voltage_limit, hub_analog_input, Tpz._RESERVED, Tpz._RESERVED
```
Use class constants.